whylog/config/investigation_plan.py

```diff
@@ -33,15 +33,25 @@ class InvestigationStep(object):
     This class is responsible for finding all possible Clues from parsed logs.
     Also controls searched time range in logs file.
     """
+    LT, EQ, GT = six.moves.range(-1, 2)
+    LEFT_BOUND, RIGHT_BOUND = six.moves.range(-1, 1)
```

Comment:
This consts should be moved into another class. It shouldn't be a part of `InvestigationStep`